test/linux/slaveinfo/slaveinfo.h

```diff
@@ -0,0 +1,12 @@
+/** \file
+ * \brief Declaration of slaveinfo for SOEM.
+ */
+
+#ifndef TEST_LINUX_SLAVEINFO_SLAVEINFO_H_
```

Comment:
SLAVEINFO_H should be enough? Would avoid having to rename the guard if the file is moved for any reason.